hrms/hr/doctype/interview/interview.py

```diff
@@ -217,8 +217,9 @@ def send_daily_feedback_reminder():
 	interview_feedback_template = frappe.get_doc(
 		"Email Template", reminder_settings.feedback_reminder_notification_template
 	)
+
 	interviews = frappe.get_all(
-		"Interview", filters={"status": ["in", ["Under Review", "Pending"]], "docstatus": ["!=", 2]}
+		"Interview", filters={"status": "Under Review", "docstatus": ["!=", 2], "scheduled_on": [ "<=", getdate(today())], "to_time": [ ">", get_time(now())]}
```

Comment:
```suggestion
		"Interview", filters={
			"status": "Under Review", 
			"docstatus": ["!=", 2], 
			"scheduled_on": ["<=", getdate()], 
			"to_time": [ "<=", now_time()]
		}
```
Fix the query:

`getdate()` returns the current date, You don't need to pass `today()` again
Again your current condition for time is still incorrect. 
`to_time > now_time()`

This will consider interviews that are ongoing or pending as per time. Reminder has to be collected for interviews that are already over.